OloEditor/src/Panels/InputSettingsPanel.cpp

```
@@ -133,8 +134,14 @@ namespace OloEngine
         ImGui::SameLine();
         if (ImGui::Button("Reset to Empty"))
         {
+            auto oldMap = InputActionManager::GetActionMap();
             InputActionManager::SetActionMap({});
             m_Dirty = true;
+            if (m_CommandHistory)
```

Clang (14.0.6)

[warning] implicit conversion 'OloEngine::CommandHistory *' -> bool

(readability-implicit-bool-conversion)